stablehlo/conversions/tosa/tests/BUILD.bazel

```diff
@@ -27,8 +27,8 @@ expand_template(
     substitutions = {
         "@LIT_SITE_CFG_IN_HEADER@": "# Autogenerated, do not edit.",
         "@LLVM_TOOLS_DIR@": package_path("@llvm-project//llvm:BUILD"),
-        "@STABLEHLO_TOOLS_DIR@": ".",
-        "@STABLEHLO_SOURCE_DIR@": ".",
+        "\"@STABLEHLO_TOOLS_DIR@\"": "os.path.join(os.environ['TEST_SRCDIR'], 'stablehlo')",
```

Comment:
`TEST_SRCDIR` is defined as "absolute path to the base of the runfiles tree".

Is this always the path where `WORKSPACE.bazel` resides? Wondering if this value changes depending on the importing project.

Reply:
The best part about this change is that this path automatically adapts depending on which WORKSPACE is picked. IOW it supports both standalone stablehlo bazel builds, and where stablehlo is instantiated as an external http_archive in a downstream monorepo. I've tested this using the Cruise monorepo and I can confirm with this change all lit tests work from there.